lemmy.yml

```diff
@@ -241,15 +243,8 @@
         enabled: true
         state: started
 
-    # - name: Change the working directory to /opt
-    #   ansible.builtin.shell:
-    #     cmd: find .                    # To list files under /opt directory
-    #     chdir: /opt                    # changes to /opt directory
-    #     register: shell_output
-    # - debug: var=shell_output
-
     - name: Start docker-compose
-      community.docker.docker_compose:
+      community.docker.docker_compose_v2:
         project_src: "{{ lemmy_base_dir }}/{{ domain }}"
         state: present
         pull: true
```

Comment:
As noted in https://github.com/LemmyNet/lemmy-ansible/pull/240 the `pull` parameter is no longer a boolean, so this value has to change from `true` to something like `always` or `missing` (it fails if the value is `true`). I don't know if this was a recent change on the Ansible side, but it tripped me up a couple of days ago.